presto-docs/src/main/sphinx/admin/spill.rst

```diff
@@ -89,6 +89,19 @@ implementation as exchange compression when they are sufficiently compressible.
 Enabling this feature can reduce the amount of disk IO at the cost
 of extra CPU load to compress and decompress spilled pages.
 
+Spill Encryption
+----------------
+
+When spill encryption is enabled (``spill-encryption-enabled`` property in
+:ref:`tuning-spilling`), spill contents will be encrypted with a randomly generated
+(per spill file) secret key. Enabling this will decrease the performance of spilling
```

Comment:
Also, this will increase latency of the whole spill operation, so decreasing `experimental.memory-revoking-threshold` should be considered when encryption is enabled.

Reply:
Will change the phrasing and add a note about `experimental.memory-revoking-threshold`.